lib/web_console/templates/console.js.erb

```diff
@@ -45,13 +45,23 @@ function CommandStorage() {
 }
 
 function Autocomplete(words, prefix) {
-  this.words = words.concat(); // to clone object
-  this.words.sort(); // by alphabetically order
+  this.words = betterWords(words);
   this.current = -1;
   this.left = 0; // [left, right)
   this.right = words.length;
   this.confirmed = false;
 
+  function betterWords(words) {
```

Comment:
Can we drop the duplicated names in the server? I think an `uniq!` call in `Context#extract` can do.